bin/parse_go_sum

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+parsed_file = File.read(ARGV[0])
+
+dependencies = []
+
+parsed_file.each_line do |line|
+  # Split on space
+  # Example - github.com/google/go-cmp v0.5.3/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
+  # Validate 3 parts - name, version and checksum
+  parts = line.split
+  if parts.length == 3
+    names = parts[0].split("/")
+    dependencies.append(
+      {
+        "namespace" => names[0..-2].join("/").to_s,
+        "name" => names[-1].to_s,
+        "version" => parts[1].to_s.gsub('/go.mod', ''),
+        "checksum" => parts[2].to_s.gsub('h1:', '')
+      }
+    )
+  end
+end
+
```

Comment:
If I'm not mistaken, shell_return looks for specific status exit codes, so I think you should add the exit codes here.

Reply:
Is there an existing library that parses go dependency files like this?

Reply:
Let me add that.